terminal-dashboard/table.bash

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+
+LIBS=(
+  "ignition-cmake2"
+  "ignition-math6"
+  "ignition-utils1"
+  "ignition-tools"
+  "ignition-common4"
+  "ignition-msgs7"
+  "ignition-transport10"
+  "ignition-fuel-tools5"
+  "ignition-plugin"
+  "ignition-rendering5"
+  "ignition-physics4"
+  "ignition-sensors5"
+  "ignition-gui5"
+  "ignition-gazebo5"
+  "ignition-launch4"
+  "ignition-edifice"
```

Comment:
It would be nice to support any collection here, maybe passing a command line argument. We may even be able to leverage `gazebodistro` so we don't need to hardcode the versions.